.coderabbit.yml

```
@@ -11,5 +11,78 @@ reviews:
+        react/jsx-pascal-case: error
+        react/jsx-filename-extension: [error, { extensions: ['.tsx', '.jsx'] }]
+        react/prop-types: error
+        react/jsx-key: error
+        react/no-array-index-key: error
+        react/no-unused-prop-types: error
+        react/jsx-no-duplicate-props: error
+        react/jsx-no-undef: error
+        react/no-direct-mutation-state: error
+
+        # DRY and clean code
+        max-lines-per-function: [error, { max: 50 }]
```

yamllint (1.35.1)

[error] too many spaces inside braces

(braces)